libs/design/src/atoms/loading-icon/loading-icon.component.spec.ts

```diff
@@ -0,0 +1,49 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { DaffLoadingIconComponent } from './loading-icon.component';
+import { Component, DebugElement } from '@angular/core';
+import { By } from '@angular/platform-browser';
+
+@Component({ template: '<daff-loading-icon class="host-element" [diameter]="diameter"></daff-loading-icon>' })
+class WrapperComponent {
+  diameter = 100;
+}
+
+describe('DaffLoadingIconComponent', () => {
+  let wrapper: WrapperComponent;
+  let component: DaffLoadingIconComponent;
+  let de: DebugElement;
+  let fixture: ComponentFixture<WrapperComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [ 
+        WrapperComponent,
+        DaffLoadingIconComponent
+      ]
+    })
+    .compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(WrapperComponent);
+    wrapper = fixture.componentInstance;
+    de = fixture.debugElement.query(By.css('daff-loading-icon'));
+    component = fixture.debugElement.query(By.css('daff-loading-icon')).componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(wrapper).toBeTruthy();
+  });
+
+  it('can take a `diameter` as input which sets max-width of the `daffloading-icon` host', () => {
+    wrapper.diameter = 50;
+    fixture.detectChanges();
+    expect(de.nativeElement.style.maxWidth).toEqual("50px");
+  });
+
+  it('has a default value of 100 for the diameter', () => {
```

Comment:
default should be 60

Reply:
Curious, I'd expect this test to fail. Will investigate.

Reply:
it's because you set it in the test, so you're not actually testing the default case

Reply:
You set it above